plugins/woocommerce/src/Admin/API/OnboardingThemes.php

```diff
@@ -260,7 +258,7 @@ public function get_recommended_themes( $request ) {
 				'color_palettes' => array(),
 				'total_palettes' => 0,
 				'slug'           => 'tsubaki',
-				'is_active'      => 'tsubaki' === $current_theme_slug,
+				'is_active'      => false,
```

Comment:
Nitpick: we could remove `is_active` from here now, to simplify things as it isn't the source of truth anymore.

Reply:
Good suggestion; I meant to remove them after testing to make sure we were overriding them correctly, and forgot to remove them later. Thanks!